support/util/util.go

```diff
@@ -141,3 +143,20 @@ func decompress(r io.Reader) (*bytes.Buffer, error) {
 
 	return bytes.NewBuffer(data), nil
 }
+
+func IsHostnameReachable(hostname string) bool {
+	port := ""
+	if strings.Contains(hostname, ":") {
```

Comment:
if this split is possible then what the func is receiving is not a hostname, so we should rename the func and/or parameter to something else e.g url.

That said this is receiving the output of ServiceExternalDNSHostname which must be a hostname.
It won't have :port.

Reply:
changed the func name.